### DISCUSSION TITLE
Custom caching is a bit weak, when will it be enhanced.

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2354] [Q&A]
After I set the cache time to a long time, Mikro could not get the latest data after modifying the data

**Comment by B4nan** [comment]
Please dont use vague terms, what does "a bit weak" mean? What would you change? Suggest concrete things if you see a room for improvement.

**Reply by Lydanne** [reply]
I'm sorry. It's my problem. I want to implement automatic caching when I find, and help me delete the cache when I modify it, but custom caching is not supported.

**Reply by klausXR** [reply]
You can create custom repositories and implement an entirely application-specific caching
